apisix/plugins/kafka-logger.lua

```diff
@@ -255,7 +264,7 @@ function _M.log(conf, ctx)
         entry = log_util.get_log_entry(plugin_name, conf, ctx)
     end
 
-    if batch_processor_manager:add_entry(conf, entry) then
+    if batch_processor_manager:add_entry(conf, entry, max_pending_entries) then
         return
```

Comment:
1. These need to be handled by the batch processor and not some plugin. This is a generic capability.

Reply:
2. If all the other batch processor configurations are in `conf`, why add a special configuration and place it in the plugin metadata?